rpc/api/eth.go

```diff
@@ -288,6 +290,43 @@ func (self *ethApi) SendRawTransaction(req *shared.Request) (interface{}, error)
 	return v, nil
 }
 
+type Transaction struct {
```

Comment:
Should we perhaps name this differently to avoid confusion with other Transaction types. I understand it's not in the types package but new developers looking through the codebase might not understand this. It's also not really a Transaction but more of placeholder for the Transaction data.